src/components/Card.js

```diff
@@ -101,7 +104,44 @@ export default class Card extends React.Component {
   }
 
   state = {
-    showDetails: false
+    showDetails: false,
+    isBookmarked: this.loadBookmarkedCards(),
+    cardID: this.loadCardID()
+  }
+
+  saveBookmarkedCards() {
+    localStorage.setItem(
+      ['isBookmarked-' + this.props.id],
+      JSON.stringify(this.state.isBookmarked)
+    )
+  }
+
+  loadBookmarkedCards() {
+    try {
+      return (
+        JSON.parse(localStorage.getItem('isBookmarked-' + this.props.id)) ||
+        false
+      )
+    } catch (err) {
+      return console.log(err) && false
+    }
+  }
+
+  saveCardID() {
+    localStorage.setItem(
+      ['cardID-' + this.props.id],
```

Comment:
Same here - I'd change the naming order. That way your data points would show up ordered by card-ID in localStorage.